pypistats/cli.py

```diff
@@ -103,15 +103,36 @@ def _valid_yyyy_mm(date_string):
     [
         argument("package"),
         argument("-p", "--period", choices=("day", "week", "month")),
-        argument("-j", "--json", action="store_true", help="Output JSON"),
+        argument(
+            "-j",
+            "--json",
+            action="store_true",
+            help='Output JSON. Shortcut for "-f json".',
+        ),
+        argument(
+            "-f",
+            "--format",
+            default="markdown",
+            help="The format of output. Supported: "
+            f'{", ".join(pypistats.FORMATS)}. '
+            'Default is "markdown".',
+        ),
```

Comment:
Let's add `choices=pypistats.FORMATS` so we don't need to list the supported formats in `help`.

Compare with `--period`.

```
optional arguments:
  -h, --help            show this help message and exit
  -p {day,week,month}, --period {day,week,month}
  -j, --json            Output JSON. Shortcut for "-f json".
  -f FORMAT, --format FORMAT
                        The format of output. Supported: json, markdown.
                        Default is "markdown".
```

Reply:
Alright. Missed that. :man_facepalming: